scripts/treedb_kahip_partition.py

```
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+"""Pinned offline KaHIP 3.25 adapter for TreeDB's validated JSON contract."""
+import json, os, sys
+# KaHIP's Python extension reads OpenMP settings during import. Force a single
+# worker so its offline output identity does not depend on host scheduling.
+os.environ["OMP_NUM_THREADS"] = "1"
+import kahip
+
+if len(sys.argv) != 3 or kahip.__version__ != "3.25":
+    raise SystemExit("requires kahip Python module version 3.25")
+with open(sys.argv[1], encoding="utf-8") as f: a = json.load(f)
```

Ruff (0.16.0)

[error] Multiple statements on one line (colon)

(E701)